index.html

```diff
@@ -412,28 +364,6 @@ <h3>The <dfn>observe()</dfn> method</h3>
         <li>
           If |target| is not equal to "`cpu`", abort these steps.
```

Comment:
Should you link to enum ComputePressureTarget for "`cpu`"?

Reply:
Ping

Reply:
This was done `{{ComputePressureTarget/cpu}}`